addons/account/static/src/xml/vat_activity.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<templates>
```

Comment:
*Not related to this, but I noticed it while reviewing this:*

The style on the dashboard seems even more fcked up than in master: all the content of the cards seems pushed to the bottom
![image](https://user-images.githubusercontent.com/44770049/186871359-9d9f3a7c-09ec-4f29-b9bc-1397a0dfc2d4.png)

Reply:
Not sure if caused by OWL or by our changes.

Reply:
Fixed